addon/components/ember-tooltip-base.js

```diff
@@ -329,10 +328,12 @@ export default Component.extend({
       placement: this.get('side'),
       title: '<span></span>',
       trigger: 'manual',
-      template: `<div class="tooltip ${tooltipClassName} ember-tooltip-effect-${this.get('effect')}" role="tooltip" style="margin:0;margin-${getOppositeSide(this.get('side'))}:${this.get('spacing')}px;">
-                  <div class="tooltip-arrow ember-tooltip-arrow"></div>
-                  <div class="tooltip-inner" id="${this.get('wormholeId')}"></div>
-                  </div>`,
+      arrowSelector: `.${emberTooltipArrowClass}`,
```

Comment:
If `emberTooltipArrayClass` is something like `ember-tooltip my-tooltip`, it should probably be turned into `.ember-tooltip.my-tooltip. Right now, it seems like it would end up being `.ember-tooltip my-tooltip`

Reply:
Good catch.